content/blog/01_mapmaking.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Map Making"
+date = 2021-05-01
+
+extra.image = "assets/images/blog/mapmaking.png"
++++
+
+Map Making 101
+
+When a builder decides that they want to make a map, they first of all, head to our (whitelisted) build server found at <code>build.nucleoid.xyz</code>
+
+They choose a game and create a map using a handy command.
+
+<br>
+<img class="header_image" src="https://iili.io/Kg34kl.jpg">
+
+One of our builders is creating a [KotH (King of the Hill)](https://github.com/NucleoidMC/koth) map using [Plasmid](https://github.com/NucleoidMC/plasmid). Plasmid is our custom game library which allows us to create minigames, and maps for those minigames, with ease.
+
+Plasmid allows our builders to access a <code>/map</code> command. The map command allows the swift creation of maps, editing, and exporting for use in production. 
+
+Each map requires regions (Unless the game doesn't require any set regions) which can be configured using the <code>/map region</code> command.
+
+For KotH, we require two vital regions:
+<br><br>
+- Spawn - Where the players will spawn after being knocked off or at the start of the game.
+
+- Throne - Where the "top of the hill" is considered.
+
+<br>
+
+For showcase purposes, our builder has set the regions to visible:
+
+<br>
+<img class="header_image" src="https://iili.io/Kg3ZBe.png">
+
+"But what do the colors mean?" you might say;
+<br><br>
+- Red - The bounding box, anything built outside this red cube will not be saved.
+- Blue - Regions, the lower region is the spawn and the topmost region is the throne.
+
+<br>
+
+By the looks of it, our builder has finished their map. 
+All they need to do is run <code>/map export</code> and their map will be ready to download at [the file system found here.](https://build.nucleoid.xyz/plasmid/export)
```

Comment:
Might be good to add somewhere a link to the Plasmid wiki [creating a map documentation](https://github.com/NucleoidMC/plasmid/wiki/Creating-a-Map) as an out-of-interest extra reading thing.